cppfd/parallel_mesh.h

```diff
@@ -28,6 +50,7 @@ class ParallelMesh
     uint64_t get_n_points_y() const { return this->n_cells_y + 1; }
     double get_cell_size() const { return this->h; }
     std::array<double,2> get_origin() const {return this->origin; }
+    uint8_t get_location_type() const { return this->location_type; }
```

Comment:
Return type does not match `    int8_t location_type`.
```suggestion
    int8_t get_location_type() const { return this->location_type; }
```